src/Symfony/Component/Security/Http/EventListener/IsGrantedAttributeListener.php

```diff
@@ -29,6 +31,7 @@ class IsGrantedAttributeListener implements EventSubscriberInterface
 {
     public function __construct(
         private AuthorizationCheckerInterface $authChecker,
+        private ?ExpressionLanguage $expressionLanguage = null,
```

Comment:
The type here should be the one from the ExpressionLanguage component (the one in `Symfony\Component\Security\Core\Authorization` is just a shortcut registering autimatically a function provider with authorization-related functions.

Reply:
Done